### PR TITLE
Capitalize "Terraform" in docs/gettings_started/clc

### PR DESCRIPTION
**Pull request by johnswil**
Hi, thank you for your contribution to Mantl! Before we can accept any code into
master, we need it to meet the following criteria. If there are any you can't
satisfy yourself, go ahead and open the pull request anyway and we'll help you
test. Feel free to delete this message once you're done. Thanks again!

- [ ] Installs cleanly on a fresh build of most recent master branch
- [ ] Upgrades cleanly from the most recent release
- [ ] Updates documentation relevant to the changes

### CODE REVIEW

File: docs/getting_started/clc.rst [hunk 1]
```diff
@@ -98,9 +98,9 @@ Additional settings are available for customization in
 Provisioning
 ------------
 
-Once you've reviewed and/or modified the settings, ``terraform get``
-will prepare your cluster, ``terraform plan`` can be reviewed to check
-the deployment, and ``terraform apply`` will provision the
+Once you've reviewed and/or modified the settings, ``Terraform get``
+will prepare your cluster, ``Terraform plan`` can be reviewed to check
```
Comment (siddharthist) [inline docs/getting_started/clc.rst]: There are commands, and shouldn't be capitalized. They won't work. 
```
$ Terraform version
zsh: command not found: Terraform
$ terraform version
Terraform v0.6.16
```

File: docs/getting_started/clc.rst [hunk 2]
```diff
@@ -4,15 +4,15 @@ CenturyLinkCloud
 .. versionadded:: 1.0.3
 
 Terraform can use CLC to provision hosts for your cluster. You
-can `download Terraform from terraform.io
+can `download Terraform from Terraform.io
```
Comment (siddharthist) [inline docs/getting_started/clc.rst]: This is a link, and shouldn't be capitalized.

**Comment by sehqlr** [comment]
@johnswil: are you able to make the edits that @siddharthist is suggesting? I can help, if need be.

**Comment by johnswil** [comment]
Yes, all have been addressed

**Comment by sehqlr** [comment]
Hm, I only see one commit

**Comment by siddharthist** [comment]
@johnswil If you have any work that addresses the comments, we can't see it.

**Comment by siddharthist** [comment]
@johnswil Are you still planning on fixing this?

**Comment by siddharthist** [comment]
@johnswil The commands still have to be de-captialized.

**Comment by siddharthist** [comment]
I'm closing this due to lack of changes. If you can fix our comments, please reopen @johnswil

**Pull request closed without merging**